Assets/HoloToolkit-Examples/Input/Scripts/XboxControllerHandlerTest.cs

```diff
@@ -25,9 +25,8 @@ public class XboxControllerHandlerTest : XboxControllerHandlerBase
         private Vector3 newPosition;
         private Vector3 newRotation;
 
-        protected override void Start()
+        private void Start()
```

Comment:
We should probably make this `protected virtual` in case someone wants to add onto the functionality of this class

Reply:
Since it's a test class, that seems unlikely, but I'll change it.

Reply:
Good point